install/hoarder-install.sh

```diff
@@ -0,0 +1,170 @@
+msg_ok "Installed Meilisearch"
+
+msg_info "Installing Node.js"
+mkdir -p /etc/apt/keyrings
+curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg
+echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_22.x nodistro main" >/etc/apt/sources.list.d/nodesource.list
+$STD apt-get update
+$STD apt-get install -y nodejs
+msg_ok "Installed Node.js"
+
+msg_info "Installing Hoarder"
+cd /opt
+RELEASE=$(curl -s https://api.github.com/repos/hoarder-app/hoarder/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+wget -q "https://github.com/hoarder-app/hoarder/archive/refs/tags/v${RELEASE}.zip"
+unzip -q v${RELEASE}.zip
+mv hoarder-${RELEASE} /opt/hoarder
+cd /opt/hoarder
+corepack enable
+export PUPPETEER_SKIP_DOWNLOAD="true"
+export NEXT_TELEMETRY_DISABLED=1
+export CI="true"
+cd /opt/hoarder/apps/web
+$STD pnpm install --frozen-lockfile
+$STD pnpm exec next build --experimental-build-mode compile
+cp -r /opt/hoarder/apps/web/.next/standalone/apps/web/server.js /opt/hoarder/apps/web
+cd /opt/hoarder/apps/workers
+$STD pnpm install --frozen-lockfile
+
+HOARDER_SECRET=$(openssl rand -base64 36 | cut -c1-24)
+cat <<EOF >/opt/hoarder/.env
+SERVER_VERSION=$RELEASE
+NEXTAUTH_SECRET="$HOARDER_SECRET"
+NEXTAUTH_URL="http://localhost:3000"
+DATA_DIR="/opt/hoarder_data"
+MEILI_ADDR="http://127.0.0.1:7700"
+MEILI_MASTER_KEY="$MASTER_KEY"
+BROWSER_WEB_URL="http://127.0.0.1:9222"
+EOF
+echo "${RELEASE}" >"/opt/${APPLICATION}_version.txt"
+msg_ok "Installed Hoarder"
+
+msg_info "Running Database Migration"
+mkdir -p /opt/hoarder_data
+cd /opt/hoarder/packages/db
+$STD pnpm migrate
+mv db.db /opt/hoarder_data
+msg_ok "Database Migration Completed"
+
+msg_info "Creating Services"
+cat <<EOF >/etc/systemd/system/meilisearch.service
+[Unit]
+Description=Meilisearch
+After=network.target
+
+[Service]
+ExecStart=/usr/bin/meilisearch --config-file-path /etc/meilisearch.toml
+Restart=always
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+cat <<EOF >/etc/systemd/system/hoarder-web.service
+[Unit]
+Description=Hoarder Web
+Wants=network.target hoarder-workers.service meilisearch.service
+After=network.target hoarder-workers.service meilisearch.service
+
+[Service]
+ExecStart=pnpm start
+WorkingDirectory=/opt/hoarder/apps/web
+EnvironmentFile=/opt/hoarder/.env
+Restart=always
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+cat <<EOF >/etc/systemd/system/hoarder-browser.service
+[Unit]
+Description=Hoarder Headless Browser
+After=network.target
+
+[Service]
+User=root
+ExecStart=/usr/bin/chromium --headless --no-sandbox --disable-gpu --disable-dev-shm-usage --remote-debugging-address=127.0.0.1 --remote-debugging-port=9222 --hide-scrollbars
+Restart=always
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+cat <<EOF >/etc/systemd/system/hoarder-workers.service
+[Unit]
+Description=Hoarder Workers
+Wants=network.target hoarder-browser.service
```

Comment:
The worker should depend on `meilisearch.service` as it's the worker that indexes the data in meillisearch.